bootloader/lk.go

```diff
@@ -148,7 +397,13 @@ func (l *lk) ExtractKernelAssets(s snap.PlaceInfo, snapf snap.Container) error {
 		return err
 	}
 
-	if l.inRuntimeMode {
+	if l.prepareImageTime {
+		// we are preparing image, just extract boot image to bootloader directory
+		logger.Debugf("ExtractKernelAssets handling image prepare")
```

Comment:
bit undecided about keeping or dropping this log. If we are to keep it, perhaps we should add some useful information:
```
logger.Debugf("extract kernel assets from %s to %s", env.GetBootImageName(), l.dir())
```

Reply:
yes I left most of these Debugf's that were already there, I agree as they currently exist they are not very useful. I will do a pass over all these leftover messages

Reply:
ah actually forgot to do this in my most recent update, I will maybe look at these in a followup

Reply:
done in https://github.com/snapcore/snapd/pull/9756